cognition_objects/strategy.py

```diff
@@ -32,7 +32,7 @@ def get_all_by_project_id(project_id: str) -> List[CognitionStrategy]:
     return (
         session.query(CognitionStrategy)
         .filter(CognitionStrategy.project_id == project_id)
-        .order_by(CognitionStrategy.created_at.asc())
+        .order_by(CognitionStrategy.order.desc())
```

Comment:
lets also order (after order) by created_at desc. to ensure we still have the same order for strategies that don't have the field filled